app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+  "Go to the National health service website": "Ir para o website da Direção Geral de Saúde",
+  "Report status by zip code": "Relatórios por código postal",
+  "What is my Zip code?": "Qual é o meu código postal?",
+  "https://www.whatismyzip.com": "https://www.whatismyzip.com",
+  "The markers correspond to coordinates of zip codes, they have nothing to do with individuals.": "Os marcadores correspondem a coordenadas de código postal e não de indivíduos.",
+  "Healthy": "Saudáveis",
+  "Has symptoms": "Com sintomas",
+  "Tested positive": "Positivos",
+  "Zip code information": "Zip code information",
+  "Join us in the most important crowdsource!": "Junte-se a nós no crowsource mais importante!",
+  "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.": "Make your voluntary contribution to the health care system and register your health condition now. Together we can create an overview.",
+  "Share on": "Partilhe no",
+  "NB!": "Importante!",
+  "This page is currently not affiliated with the authorities in any way.": "Esta página não está de forma alguma afiliada a nenhuma autoridade.",
+  "Read how the data you enter is processed here!": "Leia sobre como os seus dados são processados aqui!",
+  "Use the link you noted down last time to update your health condition": "Utilize o endereço que guardou da última vez para atualizar o seu estado de saúde.",
+  "Unreported cases": "Casos não reportados",
+  "The spread of COVID-19 in our country is unknown. Help us create a better overview.": "A extensão da propagação do COVID-19 no nosso país é desconhecida. Ajude-nos a criar uma melhor visão geral.",
+  "Contribute anonymously": "Totalmente anónimo",
+  "Report your health condition anonymously": "Submeta o seu estado de saúde anonimamente.",
+  "Insight for everyone": "Informação aberta",
+  "The data is open to everyone. Look at maps and graphs, or download the data and create your own visualizations.": "Os dados são abertos a todos. Veja mapas e gráficos ou faça download dos dados e crie as suas próprias visualizações.",
+  "Everything is open": "Código aberto",
+  "The website is open source and built by volunteers. Feel free to join.": "O código-fonte do website é aberto e desenvolvido por voluntários. Você também pode fazer parte.",
+  "Total reports": "Total de submissões",
+  "Infected / Have been tested": "Infetados / Testados",
+  "In close contact with someone who's infected": "Em contacto próximo com alguém infetado",
+  "You must approve the privacy statement to report your data.": "You must approve the privacy statement to report your data.",
+  "Your state of health": "O seu estado de saúde",
+  "Report your health condition": "Submeta o seu estado de saúde",
+  "Thank you for your contribution! Use this link if your health condition changes:": "Obrigado pelo seu contributo! Utilize este endereço case o seu estado de saúde se altere:",
```

Comment:
Just to report a minor mistake:

"Utilize este endereço **caso** o seu estado de saúde se altere:"